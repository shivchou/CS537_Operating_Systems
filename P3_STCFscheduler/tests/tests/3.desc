Check completion time is updated
