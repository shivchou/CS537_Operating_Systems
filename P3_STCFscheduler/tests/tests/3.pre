./edit-makefile.sh ../solution/Makefile test_3 > ../solution/Makefile.test
cp -f tests/test_3.c ../solution/test_3.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests
