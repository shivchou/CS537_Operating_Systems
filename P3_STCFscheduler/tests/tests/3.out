XV6_TEST: scheduler : STCF
XV6_TEST: workload 1 started
XV6_TEST: workload 1 fin
XV6_TEST: Should be reached after the parent finishes
XV6_TEST: Child fin
